src/Persistence/MassTransit.Azure.Cosmos.Table/Configuration/AzureCosmosTableAuditStoreConfiguratorExtensions.cs

```diff
@@ -0,0 +1,41 @@
+namespace MassTransit.Azure.Cosmos.Table
+{
+    using System;
+
+
+    public static class AzureCosmosTableAuditStoreConfiguratorExtensions
+    {
+        /// <summary>
+        /// </summary>
+        /// <param name="configurator"></param>
+        /// <param name="connectionString"></param>
+        /// <param name="auditTableName"></param>
+        /// <param name="configureFilter"></param>
+        public static void UseAzureCosmosTableAuditStore(this IBusFactoryConfigurator configurator, string connectionString, string auditTableName,
```

Comment:
Could be additional overload with Table reference as parameter, and DRY it out please :)